finetune/util/positional_embeddings.py

```diff
@@ -33,13 +33,18 @@ def process_embeddings(name, value):
             vocab_size = input_pipeline.text_encoder.vocab_size
             word_embeddings = value[: vocab_size]
             positional_embed = value[vocab_size:]
-
-            positional_embed = process_pos_embed(
-                positional_embed, config.max_length, config.interpolate_pos_embed
-            )
-            value = np.concatenate(
-                (word_embeddings, positional_embed), axis=0
-            )
+            
+            # glove, for instance
+            if not positional_embed.shape[0]:
```

Comment:
Our code currently assumes everything has positional embeddings tacked on to the end of the text embeddings.  There's definitely a more elegant way to handle this